source/reference/configure-shell-settings.txt

```diff
@@ -46,6 +46,12 @@ Change the current setting of ``<property>`` to ``<value>``:
 
    config.set( "<property>", <value> )
 
+Reset a ``<property>`` to its default value:
```

Comment:
Perhaps "To reset a ``<property>``'s default value:"

Reply:
To me, "Reset the default value" reads like you're changing the default value of the parameter. Instead, you're setting something _to_ the default value. I've tweaked the wording here slightly.